Zoo/src/JosPlateau/AnimalsRunner2.java

```diff
@@ -0,0 +1,15 @@
+package JosPlateau;
+
+/**
+ * Created by admin-batch-a on 7/16/17.
+ */
+public class AnimalsRunner2 {
+
+    public static void main(String[] args) {
+        Animals2 myScan = new Animals2();
+        myScan.getMyList();
+        return;
```

Comment:
main method does not need a return statement. Kindly review all our examples please.